src/adapters/PeopleJSONAdapter.js

```diff
@@ -0,0 +1,20 @@
+import {Observable} from 'rxjs';
+
+import person from './../data/person';
+import PeopleAdapter from './PeopleAdapter';
+
+export default class PeopleJSONAdapter extends PeopleAdapter {
+  /**
+   * Returns an observable that emits person data
+   *
+   * @param {String} id - ID of person to get
+   * @returns {Observable<PersonObject>}
+   * @memberof PeopleAdapter
+   */
+  getPerson(id) {
+    return new Observable((observer) => {
+      if (person.id === id) observer.next(person);
+      else observer.error(new Error(`person with id:${id} does not exist`));
+    });
```

Comment:
We should complete the observable since we know we don't have any more data to send.
```suggestion
    observer.complete();
    });
```